resources/assets/js/embed.js

```diff
@@ -1,35 +1,35 @@
-window.pixlfed = {};
-window.pixlfed.config = {
+window.pixelfed = {};
+window.pixelfed.config = {
```

Comment:
This is almost certainly an API break. It's also kinda odd, since this is the brand...

Reply:
Yeah I would revert this change, just to be on the safe side